tket/src/Predicates/PassGenerators.cpp

```diff
@@ -480,14 +481,53 @@ PassPtr gen_cx_mapping_pass(
     bool delay_measures) {
   OpTypeSet gate_set = all_single_qubit_types();
   gate_set.insert(OpType::CX);
-  PassPtr rebase_pass =
-      gen_rebase_pass(gate_set, CircPool::CX(), CircPool::tk1_to_tk1);
+  PassPtr rebase_pass = gen_auto_rebase_pass(gate_set);
   PassPtr return_pass =
```

Comment:
We can remove the references to return pass here I think

Reply:
It's used below in the line
```
  PassConditions conditions = return_pass->get_conditions();
```

Reply:
...which I think is still correct, although there may be an easier way to construct these conditions.